api/src/repositories/telemetry-repositories/telemetry-manual-repository.interface.ts

```diff
@@ -0,0 +1,22 @@
+import { TelemetryManualRecord } from '../../database-models/telemetry_manual';
+
+/**
+ * Interface reflecting the telemetry manual data required to create a new manual telemetry record.
+ *
+ */
+export type CreateManualTelemetry = Pick<
+  TelemetryManualRecord,
+  'deployment2_id' | 'latitude' | 'longitude' | 'acquisition_date' | 'transmission_date'
+>;
+
+/**
+ * Interface reflecting the telemetry manual data required to update an existing manual telemetry record.
+ *
+ */
+export type UpdateManualTelemetry = Omit<TelemetryManualRecord, 'deployment2_id'>;
+
+/**
+ * Interface reflecting the telemetry manual data required to delete an existing manual telemetry record.
+ *
+ */
+export type DeleteManualTelemetry = Pick<TelemetryManualRecord, 'telemetry_manual_id' | 'deployment2_id'>;
```

Comment:
Can deployment2_id be removed now? Also is this used anywhere? Its just 1 field now, so possibly not needed?

Reply:
Oh yea, that type is deprecated now, i'll drop it.

Reply:
🏄